Sources/AWSLambdaEvents/AppSync.swift

```diff
@@ -0,0 +1,115 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2017-2020 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+public struct AppSync {
+	public struct Event: Decodable {
+		public let arguments: [String: ArgumentValue]
+		
+		public enum ArgumentValue: Codable {
+			case string(String)
+			case dictionary([String: String])
+			
+			public init(from decoder: Decoder) throws {
+				let container = try decoder.singleValueContainer()
+				if let strValue = try? container.decode(String.self) {
+					self = .string(strValue)
+				} else if let dictionaryValue = try? container.decode([String: String].self) {
+					self = .dictionary(dictionaryValue)
+				} else {
+					throw DecodingError.dataCorruptedError(in: container, debugDescription: """
+					Unexpected AppSync argument.
+					Expected a String or a Dictionary.
+					""")
+				}
+			}
+			
+			public func encode(to encoder: Encoder) throws {
+				var container = encoder.singleValueContainer()
+				switch self {
+				case .dictionary(let array):
+					try container.encode(array)
+				case .string(let str):
+					try container.encode(str)
+				}
+			}
+		}
+        public let request: Request
+        public struct Request: Decodable {
+            let headers: HTTPHeaders
+        }
+           public let source: [String: String]?
+           public let stash: [String: String]?
+
+           public let info: Info
+		public struct Info: Codable {
+			public var selectionSetList: [String]
+			public var selectionSetGraphQL: String
+			public var parentTypeName: String
+			public var fieldName: String
+			public var variables: [String: String]
+		}
+		public let identity: Identity?
+		public struct Identity: Codable {
+			public struct Claims {
+				let sub: String
+				let emailVerified: Bool
+				let iss: String
+				let phoneNumberVerified: Bool
+				let cognitoUsername: String
+				let aud: String
+				let eventId: String
+				let tokenUse: String
+				let authTime: Int
+				let phoneNumber: String?
+				let exp: Int
+				let iat: Int
+				let email: String?
+				
+				enum CodingKeys: String, CodingKey {
+					case sub, emailVerified = "email_verified", iss, phoneNumberVerified = "phone_number_verified", cognitoUsername = "cognito:username", aud, eventId = "event_id", tokenUse = "token_use", authTime = "auth_time", phoneNumber = "phone_number", exp, iat, email
+				}
```

Comment:
Would you mind formatting your code like it is here?

https://github.com/swift-server/swift-aws-lambda-runtime/blob/bfdeb75e32a9720a859531b9228b4f25966a1223/Sources/AWSLambdaEvents/DynamoDB.swift#L106-L115